README.md

```diff
@@ -47,16 +47,21 @@ To learn how to generate executables for your favorite target platform, follow t
+| `-t`      |           `--withtypeprovider`          |  False   |      `true`       | Indicates whether the `CustomTypeProvider` class should be generated (see [Customizing the strong-type binding logic](https://github.com/kontent-ai/delivery-sdk-net/blob/master/docs/customization-and-extensibility/strongly-typed-models.md#customizing-the-strong-type-binding-logic) for more info) |
+| `-s`      |           `--structuredmodel`           |  False   |      `null`       |                                                                                                   For detailed please see [section](https://github.com/kontent-ai/model-generator-net#structured-model)                                                                                                   |
+| `-f`      |           `--filenamesuffix`            |  False   |      `null`       |                                                                                                              Adds a suffix to generated filenames (e.g., News.cs becomes News.Generated.cs)                                                                                                              |
+| `-b`      |              `--baseclass`              |  False   |      `null`       |                                                                                  If provided, a base class type will be created and all generated classes will derive from that base class via partial extender classes                                                                                  |
+| `-e`      |        `--extendeddelivermodels`        |  False   |     `false`       |                                                                                                                       Indicates whether extended deliver models should be generated                                                                                                                      |
+
+#### Structured model
+
+Generates `IRichTextContent` instead of `string` for rich-text elements or `IDateTimeContent` instead of `DateTime?` for date-time elements. This enables utilizing [structured rich-text rendering](https://github.com/kontent-ai/delivery-sdk-net/blob/master/docs/customization-and-extensibility/structured-models/structured-models-rendering.md) [structured date-time rendering](https://github.com/kontent-ai/delivery-sdk-net/blob/master/docs/customization-and-extensibility/structured-models/structured-models-rendering.md). For *modular content elements* and *subpages* generates [`IContentItem`](https://github.com/kontent-ai/delivery-sdk-net/blob/master/Kontent.Ai.Delivery.Abstractions/ContentItems/IContentItem.cs) instead of `object`, the model also implements the interface `IContentItem`.  Allowed values [`RichText`, `DateTime`, `True`, `ModularContent`], as a separator you should use `,`. ⚠️ `True` parameter is **obsolete** and interprets the same value as `RichText`.
```

Comment:
I would move info about modular content to the `structured-models-rendering.md` to keep it consistent.